step.yml

```diff
@@ -1,26 +1,57 @@
 ---
-title: Artifact pull
-summary: Step to pull artifacts of a pipeline
+title: Pull Pipeline intermediate files
+summary: Pull intermediate files generated by the Pipeline Builds.
 description: |-
-  The step downloads build artifacts of a pipeline to a local folder.
+  The step downloads intermediate files (build artifacts generated by the pipeline workflows,
+  that are intended to be shared with subsequent workflows of the pipeline) to a local folder.
 
-  By default, all artifacts generated by any workflow of the pipeline are downloaded. This can be limited
-  by setting the `artifact_sources` input variable.
+  These artifacts can be uploaded by the [Deploy to Bitrise.io step](https://www.bitrise.io/integrations/steps/deploy-to-bitrise-io),
+  via specifying the *Files to share between pipeline stages* input.
+
+  Pipeline intermediate files have an environment variable key assigned,
+  after downloading the given artifact, an environment variable (with the assigned key) will point to the file's local path.
```

Comment:
When uploaded, Pipeline intermediate files automatically receive an environment variable key. After downloading the files, the environment variable key will point to the file's local path.

Reply:
> Pipeline intermediate files automatically receive an environment variable key

This is not automatic, users explicitly need to specify what key to use with the [new pipeline_intermediate_files input of the deploy-to-bitrise-io step](https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/pull/158/files#diff-6bc88361fc7fca6b922fde7f3243d0e3815d0584b5493c16dc45633b36172529R205-R216).